src/NServiceBus.Gateway/Settings/GatewaySettings.cs

```diff
@@ -103,14 +103,13 @@ public void AddSite(string siteKey, string address, string type = "http", bool l
             });
         }
 
-        /// <summary>
-        /// Adds a receive channel that the gateway should listen to.
-        /// </summary>
+        /// <summary>Adds a receive channel that the gateway should listen to.</summary>
         /// <param name="address">The channel address.</param>
-        /// <param name="maxConcurrency">Maximum number of receive connections. Default is `1`.</param>
         /// <param name="type">The channel type. Default is `http`.</param>
+        /// <param name="maxConcurrency">Maximum number of receive connections. Default is `1`.</param>
         /// <param name="isDefault">True if this should be the default channel for send operations. Default is `false`.</param>
-        public void AddReceiveChannel(string address, string type = "http", int maxConcurrency = 1, bool isDefault = false)
+        /// <param name="proxyAddress">The proxy Address to use when gateway is behind proxy, is used in reply on headers.</param>
```

Comment:
> is used in reply on headers

that part is a bit confusing to read imo

Reply:
How about

>The address to which replies should be sent should the default channel not be publicly available, eg. when running behind a proxy

Reply:
This makes me wonder if the name ProxyAddress is too tied to HTTP? Would it be relevant to other channel types like FTP?

I think this is just another sign of the design with custom channels isn't really thought out and we just have to live with it for now